cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/strategy/AbstractReplicaMovementStrategy.java

```diff
@@ -45,6 +45,11 @@ public ReplicaMovementStrategy chain(ReplicaMovementStrategy strategy) {
           return compareResult1 == 0 ? comparator2.compare(task1, task2) : compareResult1;
         };
       }
+
+      @Override
+      public String name() {
+        return current.name() + "And" + strategy.name();
```

Comment:
I am thinking the composite strategy name to be like `PostponeUrpReplicaMovementStrategyAndPrioritizeLargeReplicaMovementStrategy`. Since the class name starts with capital letter, I feel `And` make sense here.

Reply:
What if I call my class as `StrategyOfEfeAndKun`? :)
The keyword `And` can be a part of the class name, but I feel like using `space`, `comma` or any other special character may make the parsing more clear.